RadioItem/tests/RadioItem-specs.js

```diff
@@ -1,71 +1,52 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import RadioItem from '../RadioItem';
 
-import css from '../RadioItem.module.less';
-
 describe('RadioItem', () => {
 	test('should support adding text as children', () => {
-		const radioItem = mount(
-			<RadioItem>
-				Hello RadioItem
-			</RadioItem>
-		);
+		render(<RadioItem>Hello RadioItem</RadioItem>);
+		const textField = screen.getByRole('checkbox').lastElementChild;
 
 		const expected = 'Hello RadioItem';
-		const actual = radioItem.find('ItemContent').text();
 
-		expect(actual).toBe(expected);
+		expect(textField).toHaveTextContent(expected);
 	});
 
 	test('should render correct icon when not selected', () => {
-		const radioItem = mount(
-			<RadioItem>
-				Hello RadioItem
-			</RadioItem>
-		);
+		render(<RadioItem>Hello RadioItem</RadioItem>);
 
-		const expected = 0;
-		const actual = radioItem.find(`.${css.selected}`).length;
+		const expected = 983071;
```

Comment:
Leave a comment here: // decimal converted charCode of Unicode 'iconName' character